scripts/persona_driven_data_gen/README.md

```diff
@@ -0,0 +1,48 @@
+To start make sure you have your OpenAI and Anthropic API keys and have installed the libraries listed in `requirements.txt`:
```

Comment:
Maybe give it a title like 


# Persona Data Generation

This folder contains scripts on generating persona data used for preference tuning in the Tulu 3 recipe [citation, section xxxx)